src/lib/config.ts

```diff
@@ -10,7 +10,52 @@ export interface Config {
   port: number
   metricPort: number
   logLevel: string
-  pairs: ConfigPair[]
+  chains: ChainConfig[]
 }
 
-export const syncInfoFile = env.SYNC_INFO || './.syncInfo'
+interface ChainConfig {
+  bech32Prefix: string
+  chainId: string
+  gasPrice: string
+  lcdUri: string
+  rpcUri: string
+  wallets: WalletConfig[]
+  feeFilter?: PacketFee
```

Comment:
Fee filter is chain config, but the example config has it in wallet config level